src/styles/block/header.scss

```diff
@@ -0,0 +1,101 @@
+.header {
```

Comment:
Here and everywhere - keep your props in proper order.
https://9elements.com/css-rule-order/